src/collectd_dogstatsd.py

```diff
@@ -141,6 +146,11 @@ def send_points(self, metrics):
 
             sfx_metric["metric"] = metric['metric']
             sfx_metric["dimensions"] = dims_from_tags(metric['tags'])
+            if "host" not in sfx_metric["dimensions"]:
+                sfx_metric["dimensions"]["host"] = self.get_host()
+            if "host" not in sfx_metric["dimensions"]:
```

Comment:
I think you want to check that sfx_metric["dimensions"]["host"] != "" because it's guaranteed to be in it after the lines before but possibly still ""